sdk/Cargo.toml

```diff
@@ -86,6 +86,7 @@ js-sys = "0.3.57"
 [dev-dependencies]
 anyhow = "1.0.57"
 curve25519-dalek = "3.2.1"
+hex = "0.4.3"
```

Comment:
It's used to log signatures in the `demo_secp256k1_custom_many` example. This example exists to show how to load the signatures within a program, but otherwise doesn't actually do anything useful with them.